spec/rubyhub/pull_request_create.rb

```diff
@@ -1,16 +0,0 @@
-  RSpec.describe Rubyhub::Operations::PullRequest::Create do
-    let(:options) { { template: 'your_template_name' } }
-
-    describe '.call' do
-      context 'when raises error' do
-        it 'file doesnt exist' do
-          expect { described_class.call(options) }.to raise_error(ConfigFileDoesNotExistError)
```

Comment:
the only reason to remove specs -- when we remove whole class, which was covered with this exact spec... But Rubyhub::Operations::PullRequest::Create wasn't removed. Please, move this file back

Reply:
The test itself wasn't removed completely. According to suggestion of Rubocop it was moved to the appropriate folder ```spec/rubyhub/operations/pull_request/create_spec.rb```